scripts/code-gen-helpers.sh

```diff
@@ -21,7 +21,7 @@ EOH
 )
 
 generate_selfdescribe_json() {
-  $AGENT_BIN selfdescribe > $selfdescribe_json
+  ./$AGENT_BIN selfdescribe > $selfdescribe_json
```

Comment:
Please revert this and set the `AGENT_BIN` env var in the workflows instead.

Reply:
As suggested by you I have set `AGENT_BIN` in the workflow itself. 
From this now my understanding is the same function is used by `scripts/docs/add-integrations-doc-notice` so let's not affect it.